components/entry.js

```diff
@@ -7,8 +7,14 @@ import AuthorIcons from './author-icons'
 const Entry = ({ final, first, info }) => {
   let { id, number, title, authors, date, summary } = info
 
-  // TODO: do we need to show year?
-  const formattedDate = formatDate(date, { month: 'short', day: 'numeric' })
+  // TODO: properly handle separator in formatDate()
```

Comment:
maybe open an issue in `Components` for this one?

Reply:
Aha this is the change I was struggling to test via local linking! Opened a PR here: https://github.com/carbonplan/components/pull/119

Still working out how to test locally